metecho/api/serializers.py

```diff
@@ -103,26 +125,27 @@ class Meta:
             "self_guided_tour_state",
         )
 
-    def get_sf_username(self, obj) -> dict:
+    def get_sf_username(self, obj) -> Optional[str]:
         if obj.uses_global_devhub:
             return None
         return obj.sf_username
 
 
-class MinimalUserSerializer(serializers.ModelSerializer):
-    id = serializers.CharField(read_only=True)
-
+class MinimalUserSerializer(HashIdModelSerializer):
     class Meta:
         model = User
         fields = ("id", "username", "avatar_url")
 
 
-class ProjectSerializer(serializers.ModelSerializer):
-    id = serializers.CharField(read_only=True)
+class ProjectSerializer(HashIdModelSerializer):
+    slug = serializers.CharField()
+    old_slugs = StringListField(read_only=True)
     description_rendered = MarkdownField(source="description", read_only=True)
     repo_url = serializers.SerializerMethodField()
     repo_image_url = serializers.SerializerMethodField()
     has_push_permission = serializers.SerializerMethodField()
+    github_users = StringListField()
```

Comment:
@jerivas This introduced a bug I didn't notice earlier. `project.github_users` is a list of JSON objects, not a list of strings.

cc @dvdherron -- thanks for catching this.

Reply:
Thanks, working on a fix now